neutrons/SNAPRed
# Defect 2431, fixed a bunch of items for running calibration reduction ux

**Description of work**
- ~~Remove "Load Dummy"~~
	- ~~move reduction to its own screen.~~

### Diffraction Calibration UI Flow
- ~~lite toggle, default on, disabled~~
- ~~implement  setsample,  set grouping file~~
	- ~~populate the cif file dropdown (use the defined jsons)~~
- ~~remove "begin operation" button~~
- **pixelgroupingparamters call should be using user input grouping file. (Not Done)**
- ~~hide advanced json form parameters until further developed~~
- ~~make disabled buttons visibly disabled ~~

#### Assessment Step
- ~~Would you like to commence assessment?~~
- ~~pull out runnumber and cif file field~~
- ~~removed the advanced options~~
- ~~error handling~~
		- ~~shouldnt continue or disable continue button if a failure occurred due to user input


**To test:**

open the ui and muck about with the test panel, see what breaks and why.  This is mostly a ux pr so you might want to hotwire worker_pool to always treat returns as code 200



Link to EWM item

[Defect 2431](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=2431)
